spec/features/notes_on_merge_requests_spec.rb

```diff
@@ -108,14 +108,14 @@
 
         it "shows the delete link" do
           within(".note-attachment") do
-            should have_css(".js-note-attachment-delete")
+            is_expected.to have_css(".js-note-attachment-delete")
           end
         end
 
         it "removes the attachment div and resets the edit form" do
           find(".js-note-attachment-delete").click
-          should_not have_css(".note-attachment")
-          find(".current-note-edit-form", visible: false).should_not be_visible
+          is_expected.not_to have_css(".note-attachment")
+          expect(find(".current-note-edit-form", visible: false)).not_to be_visible
```

Comment:
Line is too long. [83/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.